docs/developing.rst

```diff
@@ -91,9 +91,9 @@ Development Infrastructure
     "Continuous integration" testing.
 
     Services that run frequently while the code is undergoing changes, ensuring that the codebase still installs and has the intended behavior.
-    Currently, we use a service called `Travis CI <https://travis-ci.org>`_ for this.
-    Every time we make commits to the ``master`` branch of the openff-toolkit Github repository, a set of virtual machines that mimic brand new Linux and Mac OSX computers are created, and follow build instructions specified in the repo's ``.travis.yml`` file to install the toolkit.
-    After installing the OFF toolkit and its dependencies, these virtual machines run our test suite.
+    Currently, we use a service called `GitHub Actions <https://github.com/features/actions>`_ for this.
+    Every time we make commits to the ``master`` branch of the ``openff-toolkit`` Github repository, a set of virtual machines that mimic brand new Linux and Mac OSX computers are created. These follow build instructions (``.github/workflows/CI.yml``) file to install the toolkit.
```

Comment:
Is this actually right? I thought we periodically test `master` and also test pull requests. Either way, it might be worth mentioning that all pull requests are tested before merging.

Reply:
Oh wow, that's a major oversight. Thanks!